automation/jenkins-build.sh

```diff
@@ -3,11 +3,16 @@ set -e
 
 VERSION=$(git rev-parse --short HEAD)
 ESCAPED_BRANCH_NAME=$(echo $sourceBranch | sed 's/[^a-z0-9A-Z_.-]/-/g')
+IGNORE_CACHE=0
 
-# Try pulling the old build first for caching purposes.
-docker pull resin/${JOB_NAME}:${ESCAPED_BRANCH_NAME} || docker pull resin/${JOB_NAME}:master || true
+if [ $IGNORE_CACHE -eq 1 ]; then
+	docker build --no-cache --tag resin/${JOB_NAME}:${VERSION} .
```

Comment:
It'd be good to add `--pull` to the build as well, to guarantee using the latest version of the parent image (debian).  It probably makes sense to add even when using the cache as well